WcaOnRails/spec/models/competition_spec.rb

```diff
@@ -117,18 +117,45 @@
     expect(competition).to be_invalid
   end
 
-  it "requires unapproved competition name is not greater than 32 characters" do
+  it "requires competition name is not greater than 50 characters" do
     competition = FactoryGirl.create :competition
-    competition.showAtAll = false
-    competition.name = "A really long competition name 2016"
+    competition.showAtAll = true
+    competition.name = "A really long competition name that is greater than 50 characters 2016"
```

Comment:
Please make use of FactoryGirl for setting attributes:
```ruby
competition = FactoryGirl.create :competition, showAtAll: true, name: "A really long competition name that is greater than 50 characters 2016"
```
The same goes for the other tests below =)